_sources/Sequences/Exercises.rst

```diff
@@ -1,219 +0,0 @@
-..  Copyright (C)  Brad Miller, David Ranum, Jeffrey Elkner, Peter Wentworth, Allen B. Downey, Chris
```

Comment:
@presnick @laumurph Our Assignment creator assumes that every chapter has an Exercises subchapter.  So, I hope there is a plan to restore the Exercises for this with new ones??  If we are going to suddenly change our strategy for how we present exercises to students then that will cost us some development effort.

What is the reason for some of these exercises being eliminated entirely?

Reply:
In a lot of the chapters we have pages for Exercises and for Extra Exercises, so @presnick and I had talked about redistributing problems to check your understanding sections in the subchapters. Then the exercises that were more problem-solving-oriented would stay as exercises pages. I can make/grab some problem solving exercises and make a new Exercises subchapter here. Then I can make sure all the other chapters have an Exercises subchapter if that sounds alright?

Reply:
I think the end of chapter exercises can be for practicing concepts and also problem solving.  How do you see the difference between "check your understanding" and problems that don't require a lot of problem solving but do get the student to practice with small simple programs?